fortran/src/H5_f.c

```diff
@@ -1125,3 +1125,50 @@ h5dont_atexit_c(void)
     ret_value = 0;
     return ret_value;
 }
+
+/****if* H5_f/h5free_string_array_memory_c
+ * NAME
+ *  h5free_string_array_memory_c
+ * PURPOSE
+ *  Frees internal memory allocated for a char** string array
+ * INPUTS
+ *  array_ptr - pointer to char** array
+ *  num_files - number of strings in the array
+ * RETURNS
+ *  0 on success, -1 on failure
+ * SOURCE
+ */
+int_f
+h5free_string_array_memory_c(void **array_ptr, size_t_f *num_files)
+/******/
+{
+    int    ret_value = 0;
+    char **array;
+    size_t len;
+    size_t i;
```

Comment:
it's a nitpick, but I normally put this in the for loop now

Reply:
updated.